jobs/iscsi/initialize.sh

```diff
@@ -24,18 +24,17 @@ fi
 echo `timestamp` "Started to build devstack as a threaded job"
 
 # Deploy devstack vm
-nohup /usr/local/src/cinder-ci-2016/jobs/deploy_devstack_vm.sh $hyperv01_ip $hyperv02_ip > /home/jenkins-slave/logs/devstack-build-log-$ZUUL_UUID 2>&1 &
+nohup /usr/local/src/cinder-ci-2016/jobs/deploy_devstack_vm.sh $hyperv01_ip $hyperv02_ip > /home/jenkins-slave/logs/devstack-build-log-$ZUUL_UUID-$JOB_TYPE 2>&1 &
 pid_devstack=$!
 # Deploy Windows Cinder vm
-#source /usr/local/src/cinder-ci/jobs/deploy_cinder_windows_vm.sh
 
-nohup /usr/local/src/cinder-ci-2016/jobs/build_hyperv.sh $hyperv01_ip $JOB_TYPE > /home/jenkins-slave/logs/hyperv-$hyperv01-build-log-$ZUUL_UUID 2>&1 &
+nohup /usr/local/src/cinder-ci-2016/jobs/build_hyperv.sh $hyperv01_ip $JOB_TYPE > /home/jenkins-slave/logs/hyperv-$hyperv01-build-log-$ZUUL_UUID-$JOB_TYPE 2>&1 &
```

Comment:
We're already doing this in the new 'initialize-nodes.sh' script, trying to understand the workflow.